integr8ly/tutorial-web-app
# INTLY-3692: Dashboard changes

## Motivation
https://issues.jboss.org/browse/INTLY-3692

## What
Updated the dashboard to a tabbed UI, one for the Services and one for the Solutions.

## Why
New UX design.

## Verification Steps
1. Go to the new tabs and navigate to various elements to verify that they still work.
2. Verify the status of each service is reflected correctly.
3. Verify that you can navigate to each service's console via the button.
4. Run through a couple of different solutions to verify they still work correctly. 
5. Basically, yes this was a huge dashboard change, but it also comes with a large update to the latest Patternfly libraries, so everything within the Solution Explorer realm should be tested to verify that they are working as expected.

## Checklist:

## Progress

## Additional Notes
You can verify the changes on my RHPDS server if you like:
**https://tutorial-web-app-webapp.apps.uxddev-0f8d.open.redhat.com/
admin@example.com / F1vrqCCy6lJVlCD**

Or if you prefer, you can point Solution Explorer to the following docker build on your own server:
**docker.io/mfrances17/dev-tutorial-web-app:latest**

**Screen shot of Managed Services tab:**
![Managed Services tab](https://user-images.githubusercontent.com/39063664/70084996-671b0080-15dd-11ea-825c-2b229e98ae50.png)

**Screen shot of Solutions tab:**
![Solutions tab](https://user-images.githubusercontent.com/39063664/70085007-6edaa500-15dd-11ea-9896-bbe90a93c38d.png)
